docs/release-notes/8.5.asciidoc

```diff
@@ -1,6 +1,21 @@
 [[release-notes-header-8.5.0]]
 == 8.5
 
+[discrete]
+[[release-notes-8.5.1]]
+=== 8.5.1
+
+[discrete]
+[[bug-fixes-8.5.1]]
+==== Bug fixes and enhancements
+* Fixes a bug that caused {elastic-endpoint}s (version 8.5.0) running on Linux systems with many CPUs to sometimes become unhealthy https://github.com/elastic/endpoint/issues/34[#34].
+* Fixes bug that caused incorrect alerts to display in Timeline when investigating alerts from the Detection & Response dashboard ({pull}144319[#144319]).
+* Updates the User authentication area chart so it can be opened in Lens ({pull}144011[#144011]).
+* Displays the {jira} connector icon for users with a basic license ({pull}143916[#143916]).
+* Updates the link in the machine learning rule type card to direct users towards the Elastic licensing page ({pull}143836[#143836]).
+* Turns off the option to edit machine learning rules if users have a basic license ({pull}143260[#143260]).
```

Comment:
Basically the fix also spans for all users without ML permissions. By default users under a basic license don't have ML permissions. I'd write it down like this

* Turns off the option to edit machine learning rules if users don't have ML permissions